src/main/java/org/mitre/synthea/export/FhirR4.java

```diff
@@ -2531,7 +2660,7 @@ private static void clinicalNote(Person person,
       documentReference.addContent()
           .setAttachment(reportResource.getPresentedFormFirstRep())
           .setFormat(
-            new Coding("http://ihe.net/fhir/ValueSet/IHE.FormatCode.codesystem",
+            new Coding("http://ihe.net/fhir/ihe.formatcode.fhir/CodeSystem/formatcode",
```

Comment:
This doesn't look correct to me. See http://hl7.org/fhir/R4/valueset-formatcodes.html

Despite the fact that URI contains `ValueSet` it is listed as the code system.

Reply:
This is one of the changes I made just because of the validator. With the original I was getting this warning:
`Unable to check whether code is in value set http://hl7.org/fhir/ValueSet/formatcodes: [Code System URI 'http://ihe.net/fhir/ValueSet/IHE.FormatCode.codesystem' is unknown so the code cannot be validated]`

Then if you go to http://ihe.net/fhir/ValueSet/IHE.FormatCode.codesystem in a browser it redirects you to https://profiles.ihe.net/fhir/ihe.formatcode.fhir/1.1.0/CodeSystem-formatcode.html where it says `Official URL: http://ihe.net/fhir/ihe.formatcode.fhir/CodeSystem/formatcode` and so using that gets rid of the validator warnings

Reply:
OK, I see.